packages/web/index.js

```diff
@@ -54,6 +59,14 @@ module.exports = (neutrino, opts = {}) => {
     throw new Error('The minify.style option has been removed. To enable style minification use the @neutrinojs/style-minify preset.');
   }
 
+  if (typeof options.devtool === 'string' || typeof options.devtool === 'boolean') {
```

Comment:
This will also accept `true`, which will result in a webpack schema validation error (since `devtool` can only be `string` or `false`), but that's probably the best bet since at least they'll get a nice schema validation error message.